apisix/ssl/router/radixtree_sni.lua

```diff
@@ -120,16 +121,17 @@ end
 
 -- export the set cert/key process so we can hook it in the other plugins
 function _M.set_cert_and_key(sni, value)
```

Comment:
the gm will replace the `set_cert_and_key` function, please check the influence

Reply:
Adjustments have been made, and now it is compatible with gm ssl

Reply:
I don't see the code, No changes required?